xml/System.Net.Http/HttpClient.xml

```diff
@@ -30,37 +30,10 @@
-  
- The <xref:System.Net.Http.HttpClient> also acts as a base class for more specific HTTP clients. An example would be a FacebookHttpClient providing additional methods specific to a Facebook web service (a GetFriends method, for instance). Derived classes should not override the virtual methods on the class. Instead, use a constructor overload that accepts <xref:System.Net.Http.HttpMessageHandler> to configure any pre- or post-request processing instead.
-  
- By default on .NET Framework and Mono, <xref:System.Net.HttpWebRequest> is used to send requests to the server. This behavior can be modified by specifying a different channel in one of the constructor overloads taking a <xref:System.Net.Http.HttpMessageHandler> instance as parameter. If features like authentication or caching are required, <xref:System.Net.Http.WebRequestHandler> can be used to configure settings and the instance can be passed to the constructor. The returned handler can be passed to one of the constructor overloads taking a <xref:System.Net.Http.HttpMessageHandler> parameter.  
-  
- If an app using <xref:System.Net.Http.HttpClient> and related classes in the <xref:System.Net.Http> namespace intends to download large amounts of data (50 megabytes or more), then the app should stream those downloads and not use the default buffering. If the default buffering is used the client memory usage will get very large, potentially resulting in substantially reduced performance.  
 
- Properties of <xref:System.Net.Http.HttpClient> should not be modified while there are outstanding requests, because it is not thread-safe.
+ ### Instancing
 
-   
- The following methods are thread safe:  
-  
-1.  <xref:System.Net.Http.HttpClient.CancelPendingRequests%2A>  
-  
-2.  <xref:System.Net.Http.HttpClient.DeleteAsync%2A>  
-  
-3.  <xref:System.Net.Http.HttpClient.GetAsync%2A>  
-  
-4.  <xref:System.Net.Http.HttpClient.GetByteArrayAsync%2A>  
-  
-5.  <xref:System.Net.Http.HttpClient.GetStreamAsync%2A>  
-  
-6.  <xref:System.Net.Http.HttpClient.GetStringAsync%2A>  
-  
-7.  <xref:System.Net.Http.HttpClient.PostAsync%2A>  
-  
-8.  <xref:System.Net.Http.HttpClient.PutAsync%2A>  
-  
-9. <xref:System.Net.Http.HttpClient.SendAsync%2A>  
-  
- <xref:System.Net.Http.HttpClient> is intended to be instantiated once and re-used throughout the life of an application. Instantiating an HttpClient class for every request will exhaust the number of sockets available under heavy loads. This will result in SocketException errors. Below is an example using HttpClient correctly.  
+ <xref:System.Net.Http.HttpClient> is intended to be instantiated once and re-used throughout the life of an application. HttpClient  is designed to pool connections, and re-use a connection across multiple requests. Instantiating an HttpClient class for every request will exhaust the number of sockets available under heavy loads. This will result in SocketException errors. Below is an example using HttpClient correctly.  
```

Comment:
> This will result in SocketException errors.

Though they'll be wrapped when actually emerging into user code, right? e.g. in an HttpRequestException?